lib/fndoc.nix

```diff
@@ -0,0 +1,15 @@
+{
+  docFn = {
+    description,
+    fn,
+    examples ? [],
+    example ? null
+  }:
+  let documentedFunction =
+    {
+      type = "documentedFunction";
+      __functor = self: x: fn x;
```

Comment:
I don’t know if that benchmark shows anything for nixpkgs itself. The complete evaluation of all (non-broken) package should be benchmarked, right?